src/lib/storage/frame_of_reference_column.cpp

```diff
@@ -0,0 +1,88 @@
+#include "frame_of_reference_column.hpp"
+
+#include "utils/assert.hpp"
+#include "utils/performance_warning.hpp"
+
+#include "storage/vector_compression/base_compressed_vector.hpp"
+
+namespace opossum {
+
+template <typename T, typename U>
```

Comment:
Is it common to name the template parameter that's just there for enable_if "U". Could its name in the cpp somehow reflect its purpose? I don't know what the common approach to this would be.

Reply:
Y comes after T. It doesn’t have a purpose so anything longer than a single letter is too long and would distract from the important stuff imo. I tried to google it but didn’t find anything.

Reply:
One day we will have c++ concepts and all this SFINAE stuff won’t be needed anymore :)